CONTRIBUTING.md

```diff
@@ -1,72 +1,74 @@
 # CONTRIBUTING
 
+## Signaler un problème
 
-## Objectifs
+Si vous rencontrez un problème avec l'application vous pouvez nous contacter à contact@rdv-solidarites.fr 
 
-- Mettre en prod aussi fréquemment que possible et d'autant petites releases que possible
-- Limiter le stock des "En cours" pour éviter d'avoir trop de sujets en tête en même temps
-- Chaque développeur·se doit pouvoir être autonome dans le processus de déploiement
-
-## Installation
-
-Installer :
-- Ruby 2.7.1
-- postgresql
-- Yarn : voir https://yarnpkg.com/en/docs/install
-- Foreman : voir https://github.com/ddollar/foreman
-
-```bash
-sudo -u postgres psql -c "create role MON_LOGIN_UNIX with createdb login superuser;"
-```
-
-Afin d'initialiser l'environnement de développement, exécutez la commande suivante :
+## Soumettre une mofification
 
-```bash
-bin/setup
-```
+Voici la marche à suivre pour nous soumettre une modification de code
+- Clone le projet dans votre environement
+- Crée une branche 
+- Apporter les modifications que vous souhaitez en respectant [le guide de programmation](#Guide-de-programmation) 
+- Poussez votre code sur github
+- Crée une Pull Request 
 
-Vous pouvez aussi vous créer un compte SuperAdmin dans une console Rails :
+## Guide de programmation
 
-```
-SuperAdmin.create!(email: 'email_associated_to_your_github_account@prov.com')
-```
+### Objectifs
```

Comment:
je supprimerais bien cette partie qui est plus à destination de l'équipe "coeur" qui travaille sur les tickets priorisés. Les contributeurs externes peuvent bosser sur ce qu'ils veulent :)

Reply:
J'ai supprimée la partie Objectif pour ne laisser que : le guide de programmation. avec les les conventions.

Reply:
Suite à la conversation que vous avez eu dans l'équipe Dev. Et au regard de ce qui est fait dans l'application (actuel) il me parait normal de préciser vos motivations quant à un code qui suit les convention de Rails.

Cela dis, quand vous arriverez au limite du framework Rails, je propose de passé sur des conventions de Clean code for Ruby. histoire de rester cohérent sur les bonnes pratiques. (^^).